_sections/COVID-19/en/05-this-is-not-normal-flu.md

```diff
@@ -1,25 +1,52 @@
-## This is not a normal flu
-**Not even in the best case**
+## This is not a normal flu **Not even in the best case**
 
-The World Health Organization estimate of 3.5% mortality rate is an average across age groups. 
-According to Centers for Disease Control and Prevention director Dr. Robert Redfield, [up to 25% of cases are asymptomatic, and they are infectious. 
-Covid-19 is likely three times more infectious than flu](https://www.npr.org/sections/health-shots/2020/03/31/824155179/cdc-director-on-models-for-the-months-to-come-this-virus-is-going-to-be-with-us). 
-Data from Iceland [suggests as many as 50% of cases may be asymptomatic](https://www.cnn.com/2020/04/01/europe/iceland-testing-coronavirus-intl/index.html). 
-There is very strong consensus that those aged 60+ and those with underlying conditions are the most severely impacted.
+The World Health Organization estimate of 3.5% mortality rate is an average across age groups.
+According to Centers for Disease Control and Prevention director Dr. Robert Redfield, [up to 25% of
+cases are asymptomatic, and they are infectious.  Covid-19 is likely three times more infectious
+than
+flu](https://www.npr.org/sections/health-shots/2020/03/31/824155179/cdc-director-on-models-for-the-months-to-come-this-virus-is-going-to-be-with-us).
+Data from Iceland [suggests as many as 50% of cases may be
+asymptomatic](https://www.cnn.com/2020/04/01/europe/iceland-testing-coronavirus-intl/index.html).
+There is very strong consensus that those aged 60+ and those with underlying conditions are the most
+severely impacted.
```

Comment:
What do we think of [semantic](https://sembr.org/), rather than merely length-based line breaks? In this case, "severely impacted" would still be on line 10, because it is part of the same _unit of thought_. I personally find this style leads to easier to understand GitHub diffs.

Reply:
I'd be in favor of that but I think it's also much easier to enforce at the time of writing rather than when we try to make the markdown files compliant.